indexers/schema.graphql

```diff
@@ -6,189 +6,189 @@
 The blockchains currently indexed by the hyberbridge multi-chain indexer
 """
 enum SupportedChain {
-    ETHE
-    ARBI
-    OPTI
-    BASE
-    BSC
-    POLY
-    HYPERBRIDGE
+  ETHE
```

Comment:
We've changed the state machine ids for ethereum chains, you need to update the`extractStateMachineIdFromSubstrateEventData`

Or is there a way to remove this enum and just use a string to represent state machines all through